src/app/team/team.component.ts

```diff
@@ -1,9 +1,9 @@
 import { Component, OnInit } from '@angular/core';
 
 @Component({
-  selector: 'app-team',
+  selector: 'rx-team',
```

Comment:
@btroncone i don't love changing the selectors or the conventions that the cli gives us - any good reason why?

Reply:
It's more habit than anything, I generally adjust prefix to be app specific. I'm perfectly fine with `app` though, I'll update real quick. 👍